runtime/gc_glue_java/ConfigurationDelegate.hpp

```diff
@@ -88,6 +88,9 @@ class MM_ConfigurationDelegate
 		if (extensions->alwaysCallWriteBarrier) {
 			writeBarrierType = gc_modron_wrtbar_always;
 		}
+		if (extensions->isSnapshotAtTheBeginningBarrierEnabled()) {
+			writeBarrierType = gc_modron_wrtbar_realtime;
```

Comment:
gc_modron_wrtbar_realtime should change into gc_modron_wrtbar_satb

Reply:
renamed to  barrier to gc_modron_wrtbar_satb. I'm keeping gc_modron_wrtbar_realtime in omr until the initial opnej9 changes get committed.